spec/stac-geoparquet-spec.md

```diff
@@ -69,12 +69,30 @@ To take advantage of Parquet's columnar nature and compression, the assets shoul
 
 See [Asset Object][asset] for more.
 
-## Including a STAC Collection JSON in a STAC Geoparquet Collection
+## Including one or more STAC Collection JSON in a STAC Geoparquet Collection
 
 To make a stac-geoparquet file a fully self-contained representation, you can
-include the Collection JSON in the Parquet metadata. If present in the [Parquet
-file metadata][parquet-metadata], the key must be `stac:collection` and the
-value must be a JSON string with the Collection JSON.
+include one or more Collection JSON in the Parquet metadata. If present in the [Parquet
+file metadata][parquet-metadata], the key must be `stac:collections` and the
+value must be a JSON string with the Collections JSON as an object, keyed by the collection id.
+
+Abbreviated example in JSON form:
+
+```json
+{
+    "stac:collections": "{\"collection-a\":{\"id\":\"collection-a\",...}}"
```

Comment:
I've confused myself, but do we need the escaping here? I guess the text does say that the value of `stac:collections` is a JSON string, so I guess this does look correct.